events/wc_order_events.txt

```diff
@@ -906,31 +906,130 @@ narrative_event = {
 		name = EVTOPTB_WCORD_406
 	}
 }
+# The Creation of Blood Knights Order
+namespace = WCBKO
+narrative_event = {
+	id = WCBKO.01
+	title = EVTTITLE_WCBKO_1
+	desc = EVTDESC_WCBKO_1
+	picture = GFX_evt_sunwell
+	border = GFX_event_narrative_frame_war
+	
+	is_triggered_only = yes
+	
+	major = yes
+	major_trigger = {
+		ai = no
+	}
+	show_root = yes
+	
+	immediate = {
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = no
+			religion = cult_of_sunwell
+			culture = high_elf
+			age = 250
+			attributes = {
+				martial = 10
+			}
+			trait = brilliant_strategist
+			trait = creature_high_elf
+			trait = class_paladin_7
+			trait = zealous
+		}
+		new_character = {
+			save_event_target_as = target_order_leader
+			
+			give_nickname = nick_the_sunblade
+			
+			activate_title = { title = d_blood_knights_order status = yes }
+			d_blood_knights_order = { grant_title = PREV }
+			set_defacto_liege = ROOT
+			wealth = 600
+		}
+		religion_authority = {
+			modifier = religious_order_formed
+			years = 10
+		}
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = yes
+			religion = cult_of_sunwell
+			culture = high_elf
+			age = 200
+			trait = skilled_tactician
+			trait = creature_high_elf
+			trait = class_paladin_4
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
+		create_character = {
+			random_traits = yes
+			dynasty = none
+			female = no
+			religion = cult_of_sunwell
+			culture = high_elf
+			age = 300
+			trait = creature_high_elf
+			trait = class_paladin_6
+			trait = brilliant_strategist
+		}
+		new_character = {
+		    set_defacto_liege = ROOT
+		}
```

Comment:
So, these paladins end up in Quel'thalas court, is it intended?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81822104-be85ab00-9543-11ea-9665-d2fd11779bbf.png)

</details>

Reply:
Kinda yes, Kinda no

Reply:
I wanted them to be in order court, but they appeared in Quel'Thalas court, so I thought _this is kinda logical and I won't touch it them_